packages-ts/starknet/src/network.ts

```diff
@@ -0,0 +1,122 @@
+import { ChildProcess, spawn } from 'child_process'
+import axios from 'axios'
+
+export abstract class IntegratedDevnet {
```

Comment:
`IntegratedDevnet` is not the appropriate name for this class. Maybe something like `ChildProcessManager`?